openxcframework/BluetoothManager.swift

```diff
@@ -319,14 +317,14 @@ open class BluetoothManager: NSObject,CBCentralManagerDelegate,CBPeripheralDeleg
   // Core Bluetooth has disconnected from BLE peripheral
   open func centralManager(_ central: CBCentralManager, didDisconnectPeripheral peripheral: CBPeripheral, error: Error?) {
     vmlog("in centralManager:didDisconnectPeripheral:")
-    //vmlog(error!)
+    // update UI if necessary
     let autoOn = UserDefaults.standard.bool(forKey: "autoConnectOn")
     // just reconnect automatically to the same device for now
     if peripheral == openXCPeripheral && autoOn{
       centralManager.connect(openXCPeripheral, options:nil)
       
       // notify client if the callback is enabled
```

Comment:
This commented block can be removed

Reply:
No we need this commented code as this is open source code .User only can understand the line of code for the comment.